### DISCUSSION TITLE
Grafana/Prometheus Metrics Exporting

### DISCUSSION THREAD

**Discussion by Volkor3-16** [ArchiveBox/ArchiveBox#1228] [Ideas]
I tried looking for an open issue or discussion and couldn't find anything, so here i am

I reckon it'd be really neat to see fun stats about ArchiveBox on grafana

- Archive Size
  - Grouped by domain/page/tag (so we can track most archived websites)
- Archived Page Count
  - Grouped by domain/page/tag (like above)

Honestly I only really care about those stats, but more technical stuff could be helpful for debugging/troubleshooting

- Download times
- Execution times (for chrome headless and such)
- Failure rates

tyvm <3

**Comment by pirate** [comment]
Unfortunately I don't know anything about grafana, I've used it once or twice but wouldn't know how to serve these stats.

You can definitely get some of this info with a SQL query though. 

Open to PR's to add this, but only if it's fairly simple/isolated code as I don't want to increase complexity too much.
